core/src/main/antlr4/com/databricks/labs/remorph/parsers/snowflake/SnowflakeParser.g4

```diff
@@ -3557,7 +3557,7 @@ predicate
     | expression comparisonOperator expression                                                   # predBinop
     | expression comparisonOperator (ALL | SOME | ANY) L_PAREN subquery R_PAREN                  # predASA
     | expression IS NOT? NULL                                                                    # predIsNull
-    | expression NOT? IN L_PAREN (subquery | exprList) R_PAREN                                   # predIn
+    | (L_PAREN exprList R_PAREN | expression) NOT? IN L_PAREN (subquery | exprList) R_PAREN      # predIn
```

Comment:
```antlr
  | expression NOT? IN L_PAREN (subquery | exprList) R_PAREN                     # predIn
  | L_PAREN exprList R_PAREN  NOT? IN L_PAREN (subquery | exprList) R_PAREN      # predExprListIn
  ```
  
  keep these separate with a separate visitor then there is no need to do complex logic to see whether there is an exprList or an expression. `visitpredExprListIn` will autmatically know that it is two expression lists.